ftplugin/javascript/cfi.vim

```diff
@@ -0,0 +1,181 @@
+scriptencoding utf-8
+
+" ============================================================================
+" Bootstrap
+" ============================================================================
+
+if get(g:, 'cfi_disable') || get(g:, 'loaded_cfi_ftplugin_javascript')
+    finish
+endif
+let g:loaded_cfi_ftplugin_javascript = 1
+
+" Saving 'cpoptions' {{{
+let s:save_cpo = &cpo
+set cpo&vim
+" }}}
+
+" ============================================================================
+" Options
+" ============================================================================
+
+if !exists('g:cfi_javascript_show_params')
```

Comment:
> OK, so please remove it now and open an issue or PR later?

Well, because I don't want to leave unnecessary code.
A few TODO comments are OK, but this code is not used from anywhere.